awd/wiley.py

```diff
@@ -0,0 +1,13 @@
+import requests
+
+
+headers = {
```

Comment:
the tests run okay without these headers. are they relevant?

Reply:
This is required to access the actual PDFs, 403 errors are triggered without it. I'm exploring how to mock that out

Reply:
Check for error response in the test is a good idea. But it seems that the file is still retrieved (at least from an arbitrary client) even when the headers are absent. Also, this might now be superseded by S3 logic.